### PR TITLE
Completed Part 1.

### PR DESCRIPTION
**Pull request by djduncan585**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -52,89 +52,107 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
-//  action 3:
- 
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
+//  Noun: yeast
+//  action 1: The yeast consumes sugar
+yeast.consume();
```
Comment (matkatmusic) [inline main.cpp:75]: `yeast.consumeSugar();`
choosing effective names for your functions is an important habit to develop.   You want them to express as much information as possible to the reader, while also needing the least amount of explanation possible.  
`yeast.consume()`.  What does it consume?   I have to read your comment about it to know. 
`yeast.consumeSugar();`  this tells me everything I need to know about what happens inside that function.
- [ ] revise.

File: main.cpp [hunk 2]
```diff
@@ -52,89 +52,107 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
-
-//  action 3:
+//  Noun: cat
+//  action 1: The cat sharpens its claws on the antique sofa
+cat.sharpenClawsOn(sofa);
+//  action 2: The cat vomits on the rug
+cat.vomitOn(rug);
+//  action 3: The cat positions themself under the ankles of the housewife
+//  who is carrying a basketful of laundry at the top of the stairs
+cat.trip(Jenna);
```
Comment (matkatmusic) [inline main.cpp:116]: As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.
- [ ] don't use function parameters yet.

File: main.cpp [hunk 3]
```diff
@@ -52,89 +52,107 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
-//  action 2:
-
-//  action 3:
+//  Noun: spider
+//  action 1: The spider builds a web
+spider.buildWeb();
+//  action 2: The spider envelopes a fly in silk
+spider.getFly();
+//  action 3: The spider detonates a chicken named Margie
+spider.detonate(Margie);
```
Comment (matkatmusic) [inline main.cpp:106]: don't use function parameters yet.
- [ ] revise

File: main.cpp [hunk 4]
```diff
@@ -52,89 +52,107 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
-//  action 1:
-
-//  action 2:
+//  Noun: drumMachine
+//  action 1: The drum machine plays a kick drum sound at maximum MIDI velocity
+drumMachine.kick(127);
+//  action 2: The drum machine plays a rimshot at MIDI velocity 100
+drumMachine.rim(100);
+//  action 3: The drum machine plays a crash cymbal at MIDI velocity 80
+drumMachine.crash(80);
```
Comment (matkatmusic) [inline main.cpp:134]: these actions have no verbs in their name.  
for example: `drumMachine.playKickSample();` is an accurate pseudo code representation of your plain-english action.
 - [ ] revise.

File: main.cpp [hunk 5]
```diff
@@ -52,89 +52,107 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
 
 
 #endif
 
+/*
+Chuck:
+BIG FYI HERE: I am a computer science student with three semesters of Java,
+so use your best judgement here regarding how much of the early
+general coding concepts you wish to emphasize. Thank you!
+*/
```
Comment (matkatmusic) [inline main.cpp:154]: Ah, I wish I read this before completing the review already!  Oh well, might as well learn how to respond to code reviews and implement requested changes.  Thanks for the message.  Messages like these should be put in the Pull Request description so I see it first.
- [ ] move your FYI to the pull request description

File: main.cpp [hunk 6]
```diff
@@ -52,89 +52,107 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
-//  action 2:
-
-//  action 3:
+//  Noun: robot
+//  action 1: The robot welds together my car
+robot.assemble(hondaCivic);
+//  action 2: The robot lubricates itself
+robot.doSelfMaintenance();
+//  action 3: The robot dances to electro
+robot.danceTo("Seaquake");
```
Comment (matkatmusic) [inline main.cpp:143]: - [ ] don't use function parameters yet

**Comment by djduncan585** [comment]
Changed objects and methods such that all objects have no parameters to accept.

**Pull request merged**